cdn/snippets.py

```diff
@@ -0,0 +1,77 @@
+"""This application demonstrates how to perform operations on data (content)
+when using the Google Cloud CDN (Content Delivery Network).
+
+For more information, see the README.md under /cdn and the documentation
+at https://cloud.google.com/cdn/docs.
+"""
+
+import argparse
+import base64
+<<<<<<< HEAD
```

Comment:
looks like merge conflict indicator is a leftover here

Reply:
Fixed